src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp

```diff
@@ -483,148 +501,152 @@ void jit_load_emitter::load_words_to_dword_extension(const Vmm &vmm, const Xbyak
+        h->mov(Reg64(aux_gpr_idxs[0]), tail_mask);
+        h->kmovq(k_mask, Reg64(aux_gpr_idxs[0]));
+        h->vblendmps(vmm | k_mask, vmm, table_val(fill_value));
     }
+}
 
 void jit_load_emitter::register_table_entries() {
-    push_arg_entry_of("zero", 0x00000000, true);
-    push_arg_entry_of("int_one", 0x00000001, true);
-    push_arg_entry_of("float_one", 0x3f800000, true);
-    push_arg_entry_of("int32_min", 0xcf000000, true);
-    push_arg_entry_of("float_min", 0xff7fffff, true);
-    push_arg_entry_of("int32_max", 0x4effffff, true);
-    push_arg_entry_of("float_max", 0x7f7fffff, true);
+    if (is_fill_) {
+        push_arg_entry_of("zero", 0x00000000, true);
+        push_arg_entry_of("int_one", 0x00000001, true);
+        push_arg_entry_of("float_one", 0x3f800000, true);
+        push_arg_entry_of("int32_min", 0xcf000000, true);
+        push_arg_entry_of("float_min", 0xff7fffff, true);
+        push_arg_entry_of("int32_max", 0x4effffff, true);
+        push_arg_entry_of("float_max", 0x7f7fffff, true);
+    }
 }
 
 /// STORE ///
-jit_store_emitter::jit_store_emitter(jit_generator *host, cpu_isa_t host_isa,
-    Precision exec_prc, emitter_in_out_map in_out_type)
-: jit_emitter(host, host_isa, exec_prc, in_out_type), name("unknown") {
-    v_len_elt = get_vec_length() / exec_prc.size();
+jit_store_emitter::jit_store_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t host_isa,
+                                     Precision src_prc, Precision dst_prc, int store_num, Precision exec_prc, emitter_in_out_map in_out_type)
+: jit_emitter(host, host_isa, exec_prc, in_out_type), store_num_(store_num), src_prc_(src_prc), dst_prc_(dst_prc), name_("unknown") {
+    v_len_elt_ = get_vec_length() / exec_prc.size();
+    store_size_ = store_num * dst_prc.size();
     if (!mayiuse(cpu::x64::avx512_core_bf16) && mayiuse(cpu::x64::avx512_core)) {
-        emu_vcvtneps2bf16.reset(new jit_emu_vcvtneps2bf16(host, host_isa));
+        emu_vcvtneps2bf16_.reset(new jit_emu_vcvtneps2bf16(host, host_isa));
     }
 }
 
-// 0 for temp reg for mask store
+// 0 for temp reg for mask store for avx512
 size_t jit_store_emitter::aux_gprs_count() const {
-    return 1;
+    return mayiuse(cpu::x64::avx512_core) && !one_of(store_num_, 16, 8, 4) ? 1 : 0;
 }
 
-// zero value, zeroed and passed from caller from performance standpoint(zeroed one time and not need preserve and restore status)
 size_t jit_store_emitter::aux_vecs_count() const {
-    return 1;
+    int count = 0;
+    // to avoid src vmm pollution after data type conversion
+    if ((src_prc_.is_float() && !dst_prc_.is_float()) || (!src_prc_.is_float() && dst_prc_.is_float()))
+        count++;
+    // zero value, zeroed and passed from caller from performance standpoint(zeroed one time and not need preserve and restore status)
+    if ((mayiuse(cpu::x64::avx512_core) && !one_of(store_num_, 16, 8, 4)) || one_of(dst_prc_, Precision::U8, Precision::U16))
```

Comment:
```suggestion
    if (one_of(dst_prc_, Precision::U8, Precision::U16))
```

Reply:
I think we should remove `!one_of(store_num_, 16, 8, 4)` but leave `mayiuse(cpu::x64::avx512_core)`. Because only on AVX512 we clamp `vmm` using zeroed `aux_vmm`.
Thanks!

Reply:
yep, agree. Ideally storing unsigned data(u8/u16) should be clamp with zero from safety perspective for all code pass. This clamp for ISA before than AVX512 is actually ensured by unsigned extend instructions, which have saturation internally, so do not need it for current implementation for ISA before AVX512.